docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,37 @@ parent = "smn_cli"
 
     Creates a new network with a name specified by the user
 
-      -d, --driver=      Driver to manage the Network
+      -d, --driver=[type]      Driver to manage the network
```

Comment:
I wonder if we should use `[type]` here; IIUC, this is the driver *name*, not *type*, and depends on which drivers are available